presto-orc/src/test/java/com/facebook/presto/orc/TestTupleDomainFilter.java

```diff
@@ -144,6 +144,11 @@ public void testDoubleRange()
         assertFalse(filter.testNull());
         assertFalse(filter.testDouble(-0.3));
         assertFalse(filter.testDouble(55.6));
+        assertFalse(filter.testDouble(Double.NaN));
+
+        filter = DoubleRange.of(Double.NaN, false, false, Double.NaN, false, false, false);
+        assertFalse(filter.testDouble(1.0));
+        assertFalse(filter.testDouble(Double.NaN));
```

Comment:
@bhhari Would `testDouble` ever return true for NaN? If so, could you add a test case? Same for float.

Reply:
any comparison with NaN should return false